server/server/models/user.py

```diff
@@ -1,18 +1,20 @@
 """SQLAlchemy model for table users"""
-from datetime import datetime
+
+import datetime
 
 from passlib.hash import pbkdf2_sha256
 from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
 from sqlalchemy.orm import relationship
 
+import pyramid.httpexceptions as exc
```

Comment:
I understand that `exc` is shorter than `httpexceptions`, but for readability the latter is much better:
```
from pyramid import httpexceptions
```